.azuredevops/publish-platform-nginxplus.yaml

```diff
@@ -48,6 +49,13 @@ extends:
         developmentEnvironment: true
         azureRegions:
           primary: 'UKSouth'
+      - name: 'ssv5'
+        serviceConnection: $(ssv5Subscription)
+        deploymentBranches:
+          - 'refs/heads/main'
+        developmentEnvironment: true
```

Comment:
ssv5 isn't dev env?

Reply:
ssv3 is dev

Reply:
@ganeshg2491 please remove  developmentEnvironment: true

Reply:
Updated. Also updated -AzureSubscription parameter to use $(subscriptionName) from env file.